src/normalize.js

```diff
@@ -22,6 +22,26 @@ exports.processContentType = (content_type, createNodeId) => {
     return nodeData;
 }
 
+exports.processAsset = (asset, createNodeId) => {
+    const nodeId = makeAssetNodeUid(asset, createNodeId);
+    const nodeContent = JSON.stringify(asset);
+    const nodeContentDigest = crypto
```

Comment:
We pass this exact function as `createContentDigest` to sourceNodes. To grab it:

```js
exports.sourceNodes = ({ actions, createContentDigest }) => {
  // use or pass createContentDigest
}
```

Reply:
Done